processing/src/test/java/io/druid/query/filter/BetweenDimFilterTests.java

```diff
@@ -0,0 +1,57 @@
+package io.druid.query.filter;
+
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Injector;
+import com.google.inject.Key;
+import io.druid.guice.GuiceInjectors;
+import io.druid.guice.annotations.Json;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+@RunWith(Parameterized.class)
+public class BetweenDimFilterTests
+{
+  public BetweenDimFilterTests(BoundDimFilter boundDimFilter) {this.boundDimFilter = boundDimFilter;}
+
+  private final BoundDimFilter boundDimFilter;
+
+  @Parameterized.Parameters
+  public static Iterable<Object[]>  constructorFeeder(){
+
+    return ImmutableList.of(new Object[]{new BoundDimFilter("dimension", "12", "15", null, null, null)},
+                            new Object[]{new BoundDimFilter("dimension", "12", "15", null, true, false)},
+                            new Object[]{new BoundDimFilter("dimension", "12", "15", null, null, true)},
+                            new Object[]{new BoundDimFilter("dimension", "12", "15", null, true, true)},
+                            new Object[]{new BoundDimFilter("dimension", "12", "15", true, null, null)},
+                            new Object[]{new BoundDimFilter("dimension", "12", "15", true, null, true)},
+                            new Object[]{new BoundDimFilter("dimension", "12", "15", true, true, true)},
+                            new Object[]{new BoundDimFilter("dimension", "12", "15", true, true, false)});
+  }
+
+  @Test
+  public void testSerDesBetweenFilter() throws IOException
+  {
+    Injector defaultInjector = GuiceInjectors.makeStartupInjector();
+    ObjectMapper mapper = defaultInjector.getInstance(Key.get(ObjectMapper.class, Json.class));
+    String serBetweenDimFilter = mapper.writeValueAsString(boundDimFilter);
```

Comment:
i think you mean TestUtils.getTestObjectMapper(), this will not work due to dependencies issues